### PR TITLE
Add namespace packages

### PR DESCRIPTION
**Pull request by hyejeongpark**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
 from setuptools import setup, find_packages
 
 setup(
-    name="xoxzo.cloudpy",
-    version="0.3.2",
+    name="xoxzo_cloudpy",
```
Comment (k4ml) [inline setup.py]: @hyejeongpark What the reason of this changes from `.` to under `_` ?

Comment (hyejeongpark) [reply setup.py]: @k4ml Oh, I think I read a post and misunderstood it. I changed the name again. Other than, does this look okay?

Comment (k4ml) [reply setup.py]: @hyejeongpark have you tested this by installing 2 namespaced packages - xoxzo.cloudpy and xoxzo.something ?

**Comment by k4ml** [comment]
@hyejeongpark refer here - https://packaging.python.org/guides/packaging-namespace-packages/#creating-a-namespace-package.

So if this need to support both python 2 and python 3, then our only option is to use the approach no. 2 - pkgutil-style namespace packages.

**Pull request merged**